src/Microsoft.SqlTools.ServiceLayer/Connection/ReliableConnection/SqlConnectionHelperScripts.cs

```diff
@@ -47,5 +47,10 @@ @rc 			int
 ";
 
     public const string GetOsVersion = @"SELECT OSVersion = RIGHT(@@version, LEN(@@version)- 3 -charindex (' ON ', @@version))";
+    public const string GetBigDataClusterEndpoints = @"
+SELECT [service_name]
```

Comment:
Minor: seems like convention is to put the queries on 1 line if possible. This is small enough that this seems easy to do.